examples/images_contours_and_fields/tricontour_demo.py

```diff
@@ -75,7 +75,7 @@
 y = np.degrees(xy[:, 1])
 x0 = -5
 y0 = 52
-z = np.exp(-0.01 * ((x - x0) ** 2 + (y - y0) ** 2))
+z = np.exp(-0.01 * ((x - x0)**2 + (y - y0)**2))
```

Comment:
generally (throughout the pr) it feels wrong to me to have no spaces around the power when there are spaces within the operand (`(x - x0)`).

Reply:
Given the possible alternatives
~~~
((x - x0) ** 2 + (y - y0) ** 2))

((x - x0)**2 + (y - y0)**2))

((x-x0)**2 + (y-y0)**2))
~~~
I find the given one the best; the power is an attachment to the parenthesis. Spaces everywhere do not give any guidance. No spaces in the parentheses is sort of ok if the operand is as simple as here.

Reply:
It feels rather silly to discuss this here, but https://www.python.org/dev/peps/pep-0008/#other-recommendations does say yes for `(a+b) * (a-b)` and no for `(a + b) * (a - b)`, which seems the closest to the current case (and which I agree with).

Reply:
I've removed spaces in the above case and similar cases. Rule of thumb: No spaces between most inner operators the there are multiple of these terms.

I know now why I do not want spaces around the power operator: In typeset math, you will have spacing around all regular operators, but the power superscript is directly attached to it's operand, whether it's a single variable or a bracketed term.

Reply:
To take an example further down, `(0.2 + (ym - yy) / ym)**2` really looks wrong to me (even with the parenthesis, the power really seems to bind to `ym`).

Reply:
My 2 cents: I agree with @timhoffm to remove the spaces around the power operator, especially when the term is being raised to an integer power.

Reply:
> `(0.2 + (ym - yy) / ym)**2` really looks wrong to me

Long terms are hard to read anyway and I can't argue against personal impression. So what are the options:

- Removing all the inner spaces makes it unreadble
  `(0.2+(ym-yy)/ym)**2`
- Having spaces around the power operator with short arguments detaches the exponent
  `(0.2 + (ym - yy) / ym) ** 2`
- IMHO the proposed version is still the best; and it's similar to the TeX rendered version:
   `(0.2 + (ym - yy) / ym)**2`
  ![grafik](https://user-images.githubusercontent.com/2836374/71689492-6f17bd00-2da2-11ea-81b4-e9460056bc96.png)
  give or take spaces around the division operator.

I'm not claiming to be able to solve all readability issues.

Reply:
I don't feel strongly enough to block this, but let's see what others think.